### PR TITLE
KEYCLOAK-16234 initial commit

### PR DESCRIPTION
**Pull request by briandooley**

### CODE REVIEW

**Review by andymunro — APPROVED** [review]
No changes. thanks for the rapid rework.  Once Joan provides her final user chapter, I will submit everything minus SSO protocols, then merge SSO protocols.

File: server_admin/topics.adoc [hunk 1]
```diff
@@ -16,10 +16,6 @@ include::topics/authentication/kerberos.adoc[]
 include::topics/authentication/x509.adoc[]
 include::topics/authentication/webauthn.adoc[]
 include::topics/sso-protocols.adoc[]
-include::topics/sso-protocols/oidc.adoc[]
-include::topics/sso-protocols/saml.adoc[]
-include::topics/sso-protocols/saml-vs-oidc.adoc[]
-include::topics/sso-protocols/docker.adoc[]
```
Comment (andymunro) [inline server_admin/topics.adoc:22]: Can you move these includes to the sso-protocols.adoc file?  That is the assembly where they go. 
Eventually I think that topics.adoc will only include the chapter head sections, like sso-protocols.adoc

**Review by andymunro — APPROVED** [review]
Excellent.  Looks like it follows all the rules.   I will merge once I have Joan's input on the Users chapter.

**Comment by andymunro** [comment]
@briandooley There is still one conflict.  Otherwise, I am ready to merge it.

File: server_admin/topics.adoc [hunk 2]
```diff
@@ -16,10 +16,10 @@ include::topics/authentication/kerberos.adoc[]
...
+include::topics/clients/assembly-client-oidc.adoc[]
```
Comment (andymunro) [inline server_admin/topics.adoc]: Looks like this include is in the wrong place.  It goes in the file:  clients/assembly-managing-clients.adoc.

File: server_admin/topics/sso-protocols/con-saml.adoc [hunk 3]
```diff
@@ -0,0 +1,13 @@
+[id="con-saml_{context}"]
+
+[[_saml]]
```
Comment (andymunro) [inline server_admin/topics/sso-protocols/con-saml.adoc:3]: I think there may be a problem with using two anchor ids.  Using CCutil on this caused an error, so best to remove [[_saml]] here.

You may also still have an older version of the file with that anchor called saml.adoc.

**Review by andymunro — COMMENTED** [review]
@briandooley I suspect there are old versions of files that have the same anchors as the new ones.  The book built with MVN, but not with CCutil.  Somehow CCutil still finds the older files. For example, do you still have a file sso-protocols/saml.adoc?

We can review at tomorrow's synch up if that works.

File: server_admin/topics.adoc [hunk 4]
```diff
@@ -4,7 +4,17 @@ include::topics/overview/how.adoc[]
 include::topics/overview/concepts.adoc[]
 include::topics/initialization.adoc[]
 include::topics/admin-console.adoc[]
+<<<<<<< HEAD
```
Comment (andymunro) [inline server_admin/topics.adoc]: Remove this line.

File: server_admin/topics.adoc [hunk 5]
```diff
@@ -4,7 +4,17 @@ include::topics/overview/how.adoc[]
...
 include::topics/assembly-managing-users.adoc[]
+=======
```
Comment (andymunro) [inline server_admin/topics.adoc]: Remove this line.

File: server_admin/topics/users/proc-enabling-terms-conditions.adoc [hunk 6]
```diff
@@ -3,7 +3,7 @@
 // con-required-actions.adoc
 
 [id="proc-enabling-terms-conditions_{context}"]
-= Enabling terms and conditions
+= Terms and conditions
```
Comment (andymunro) [inline server_admin/topics/users/proc-enabling-terms-conditions.adoc]: How about: "Enabling terms and conditions as a required action"

**Review by andymunro — CHANGES_REQUESTED** [review]
Just a couple of changes.

**Comment by briandooley** [comment]
Have made the requested changes.

**Review by andymunro — APPROVED** [review]

**Comment by andymunro** [comment]
Thanks, Brian. Merged.

**Pull request merged**